### PR TITLE
iOS WOPI requirements

### PR DESCRIPTION
**Pull request by gtritchie**
Copy over the wopi_requirements page content.

### CODE REVIEW

File: docs/native/src/wopi_requirements.rst [hunk 1]
```diff
@@ -1,12 +1,63 @@
...
 ..  _requirements:
 
-|stub-icon| WOPI implementation requirements for |Office iOS| integration
+WOPI implementation requirements for |Office iOS| integration
 =========================================================================
 
 This section documents specific requirements for your WOPI implementation for integration with |Office iOS| beyond
 what is documented in general for WOPI.
 
+Required WOPI Operations for Office for iOS
```
Comment (tylerbutler) [inline docs/native/src/wopi_requirements.rst]: use `|Office iOS|` placeholder.

File: docs/native/src/wopi_requirements.rst [hunk 2]
```diff
@@ -1,12 +1,63 @@
...
+-------------------------------------------
+
+The following WOPI operations are required for integration on |Office iOS|. Operations not listed here are not 
+currently called by Office for iOS.
```
Comment (tylerbutler) [inline docs/native/src/wopi_requirements.rst]: use `|Office iOS|` placeholder

File: docs/native/src/wopi_requirements.rst [hunk 3]
```diff
@@ -1,12 +1,63 @@
...
+~~~~~~~~~~~~
+
+* :ref:`Bootstrap`
+* :ref:`GetNewAccessToken`
+* :ref:`Shortcut operations` (Of these, only :ref:`GetRootContainer (bootstrapper)` is required)
+
+Future Support
+~~~~~~~~~~~~~~
+
+It is recommended the following operations also be implemented - it is expected the Office mobile integration will use these in the future.
```
Comment (tylerbutler) [inline docs/native/src/wopi_requirements.rst]: use `|Office iOS|` placeholder instead of 'Office mobile'.

**Comment by tylerbutler** [comment]
Suggestion: include a section about what Validator test categories must/should be passing for the iOS integration.

**Comment by gtritchie** [comment]
I passed along the suggestion to document Validator test categories, and changed 'Office mobile' and 'Office for iOS' to the placeholder.

**Pull request merged**